src/components/document/annotation-layer.tsx

```diff
@@ -41,12 +42,30 @@ export const AnnotationLayer = observer(function AnnotationLayer({
   const persistentUI = usePersistentUIStore();
   const tileApiInterface = useContext(TileApiInterfaceContext);
 
-  // Clear a partially completed annotation when the mode changes
+  // Clicking to select annotations
+  function handleArrowClick(arrowId: string, event: MouseEvent) {
+    event.stopPropagation();
+    const annotation = content?.annotations.get(arrowId);
+    if (annotation) {
+      if (annotation.isSelected) {
+        annotation.setSelected(false); // Toggle off
```

Comment:
In many use cases, clicking on a selected item without a modifier doesn't change the selection, while a modifier is required to toggle.

Reply:
I just updated this behavior.